src/test/java/com/example/DoughnutShopTest.java

```diff
@@ -0,0 +1,48 @@
+package com.example;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DoughnutShopTest {
+
+    @Test
+    public void testPrepareDoughnut_forChocolate() {
+        DoughnutShop shop = new DoughnutShop();
```

Comment:
Instead of instantiating a new `DoughnutShop` every time, we could share a single instance between all the tests